package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "joi-extract-type",
-  "version": "0.1.8",
+  "version": "0.2.0",
```

Comment:
@TCMiranda - not sure what you want to do about the versioning as this is a breaking change based on a move to joi 15.x? My suggestion would be to use the Joi version major as the major for this repo. So this would become `15.0.0`. That way multiple major versions can be maintained.

Reply:
Hey @guyellis, thank you for the work and the suggestion.
I don't know either but I think it is a common and reasonable approach to version a library based on its dependency.
So can you update it to 15.0.0?

Reply:
@TCMiranda - good idea - I've updated it to 15. Thanks.